### ISSUE TITLE
ensure examples all run

### ISSUE THREAD

**Issue by WilkAndy** [OrchardLANL/DPFEHM.jl#13]
It would be nice to have a script/action/whatever to run all the examples.  Preferably, this could be run on every PullRequest, or at least, manually run to check for regressions.  Currently, i'm going through all the examples, and many of them don't run without warnings/errors

`fracture_network_solver_scaling/...` often complains about "Package FEHM does not have DelimitedFiles in its dependencies..."
`gw_multigrid_inverse/inputdeck.jl:30` file not found
`piml/theislike_piml_setup.jl:73` params not defined

I haven't checked others

Refs https://github.com/openjournals/joss-reviews/issues/4560

**Comment by WilkAndy** [comment]
`pressure_management/ex.jl`   ERROR: LoadError: UndefVarError: flatten not defined at line 82

`richards/inputdeck.jl`  ERROR: LoadError: BoundsError: attempt to access 640800×640800 SparseArrays.SparseMatrixCSC{Float64, Int64} with 3200798 stored entries at index [1:640800, 640801]

`richards_steadystate_3d/inputdeck.jl`  Similar BoundsError

`scott_example/ex.jl` might be running - i killed it after about 20mins of no output.

`theislike/ex.jl`  ERROR: LoadError: DimensionMismatch("arrays could not be broadcast to a common size; got a dimension with lengths 2401 and 2601") at line 75

I believe all the other examples are working.

**Comment by omalled** [comment]
OK, I went through those examples and either got it running or deleted the example (some of there were basically deprecated by newer examples). So, I'll close the issue.

There are still some warnings, which I'm OK with. I can try to fix the warnings if you insist, but, e.g., fixing the FEHM warning requires a fix in a different package. I really appreciate you going through these examples! Those Richards examples that weren't working highlighted a testing gap, which I also fixed [here](https://github.com/OrchardLANL/DPFEHM.jl/commit/13564351cdf31822f48dc29f79769ada5306c1dc).
